lib/python/picongpu/input/parameters.py

```diff
@@ -113,15 +118,28 @@ def __init__(self, name, ptype, unit, default, slider_min, slider_max,
             The maximal value of the slider widget
         slider_step: float
             The stepsize when adjusting the slider
+        label: string [optional]
+            Overwrite the name for UI
+        formatter: function or lambda of form f(x) -> string [optional]
+            representation of the current value for UI
 
         """
-        Parameter.__init__(self, name, ptype, unit, default, value)
+        Parameter.__init__(self, name, ptype, unit, default, value, dtype)
```

Comment:
QUESTION:
`dtype`is (as far as I can tell) not used below, why is is then used as input parameter?
Add a todo if you want to add functionality using `dtype` in the future.

Reply:
dtype is used to init a member of the parent class.

it is then used in all `set_value`

Reply:
Sorry for the confusion - my fault - it is used right there